### PR TITLE
WIP: Add missing licensing information

### PR DESCRIPTION
**Pull request by kirelagin**
Based on #9.

Files without licensing information:

  * `templates/reuse/licenses/*` (hard to find info)
  * `templates/haskell/ci/travis.yml` (from universum)
  * `templates/haskell/ci/appveyor.yml` (from stack)

With the latter two I’m not sure how to properly record their licenses
without making repsitory initialisation process unnecessarily complex
(as those licenses will need to be carried around with _any_ repository
that includes those files).

### CODE REVIEW

**Comment by kirelagin** [comment]
cc @gromakovsky

**Comment by kirelagin** [comment]
Given #10, maybe it would be better to merge the Makefile fix separately 🤔

**Comment by kirelagin** [comment]
```
─› reuse lint
# MISSING COPYRIGHT AND LICENSING INFORMATION

The following files have no copyright and licensing information:
* templates/haskell/ci/appveyor.yml
* templates/haskell/ci/travis.yml


# SUMMARY

* Bad licenses:
* Missing licenses:
* Unused licenses:
* Used licenses: LicenseRef-ReplaceMe, Unlicense
* Read errors: 0
* Files with copyright information: 13 / 15
* Files with license information: 13 / 15
```

I think it will be way easier if we just Unlicense all the template files, so that anyone can do with them whatever they want, we honestly don’t care about these.
Also signed commits.

File: LICENSES/LicenseRef-ReplaceMe.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Unlicense.txt
```
Comment (gromakovsky) [inline LICENSES/LicenseRef-ReplaceMe.txt]: How does it work? Why is it done this way?

File: templates/.github/CONTRIBUTING.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ but, basically, it all boils down to the following:
     ```haskell
     {- SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
      -
-     - SPDX-License-Identifier: MPL-2.0
+     - SPDX-License-Identifier: LicenseRef-ReplaceMe
```
Comment (gromakovsky) [inline templates/.github/CONTRIBUTING.md]: I would preserve `MPL-2.0` here because it's our default choice.

File: templates/reuse/dep5 [hunk 3]
```diff
@@ -1,5 +1,8 @@
+# SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+# SPDX-License: Unlicense
+
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 
 Files: .gitlab/*
```
Comment (gromakovsky) [inline templates/reuse/dep5]: What about `.github`?

File: LICENSES/LicenseRef-ReplaceMe.txt (hunk 1 above)
Comment (kirelagin) [reply LICENSES/LicenseRef-ReplaceMe.txt]: That’s a symlink.

File: templates/reuse/dep5 (hunk 3 above)
Comment (kirelagin) [reply templates/reuse/dep5]: Well, that’s just an example of a dep5 file, and projects usually have only one of `{.github, .gitlab}`, so the choice is pretty much arbitrary. I think I just copy-pasted it from that morley PR.

File: templates/.github/CONTRIBUTING.md (hunk 2 above)
Comment (kirelagin) [reply templates/.github/CONTRIBUTING.md]: Well, I mean, this way it is guaranteed to make the person think.
(The real reason I did this was that the lint tool saw these lines, thought it was a header, and reported that it doesn’t know the MPL-2.0 license :).

File: templates/reuse/dep5 (hunk 3 above)
Comment (gromakovsky) [reply templates/reuse/dep5]: Ok, but isn't it weird that we have `.github` in templates and `.gitlab` in this file?

Comment (kirelagin) [reply templates/reuse/dep5]: Maybe 🤔. We can add a `.gitlab`template as well. 🤷‍♂

**Comment by gromakovsky** [comment]
Why does `templates/README.md` have `LicenseRef-ReplaceMe` and `templates/docs/branching.md` has `Unlicense`? I would expect to have `LicenseRef-ReplaceMe` everywhere, so that when a repository is bootstrapped people will change it the license used in their (new) repository.

**Comment by gromakovsky** [comment]
And btw, shouldn't `templates/reuse` be called `templates/.reuse`?

**Comment by kirelagin** [comment]
Because changing it in the `README` is mandatory, as this file is part of the target project, while `branching.md` and most other files are a general-purpose “serokell” file and there is no need to change anything in it after you copy it (although you are allowed to by Unlicense).

**Comment by kirelagin** [comment]
> And btw, shouldn't `templates/reuse` be called `templates/.reuse`?

Well, I don’t know. I think, it shouldn’t because you are no supposed to just copy the directory (e.g. like `.github`). This `reuse` directory is more like `haskell` directory, which contains files that go to different placed. E.g. `dep5` goes to `/.reuse/`, while licenses go to `/LICENSES/` in the root, not to `.reuse`.

**Comment by gromakovsky** [comment]
> Because changing it in the README is mandatory, as this file is part of the target project, while branching.md and most other files are a general-purpose “serokell” file and there is no need to change anything in it after you copy it (although you are allowed to by Unlicense).

Well, if we copy `make/Makefile`, shouldn't we change the license there as well? It is quite likely that it will be modified in the target project. At least because it has `mypackage` placeholder. I think other files also have good chances to be modified over time (e. g. we have `.hlint.yaml` and at some point people may decide that they want to remove some rule from there in their particular project).

**Comment by gromakovsky** [comment]
> This reuse directory is more like haskell directory, which contains files that go to different placed.

Hm, ok.

**Comment by kirelagin** [comment]
> shouldn't we change the license there as well?

For small changes like the ones you describe, again, it is not really mandatory. These kind of changes are not “legally significant” as they are very small.

**Comment by kirelagin** [comment]
Tbh, I’d argue that it is relatively hard to make a significant change to a configuration file that consist of yes/no or numbers (unlike e.g. CI configs that contain non-trivial scripts). That’s why I think it’s easier to Unlicense them and forget about them forever.

**Comment by gromakovsky** [comment]
Ok, I didn't know abut “legally significant” thing.

**Pull request merged**